packages/front-end/components/Experiment/ExperimentList.tsx

```diff
@@ -32,13 +32,17 @@ export default function ExperimentList({
           endDate;
 
         test.phases.forEach((p) => {
-          // @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'.
-          if (!startDate || p.dateStarted < startDate) {
-            // @ts-expect-error TS(2322) If you come across this, please fix it!: Type 'string | undefined' is not assignable to typ... Remove this comment to see the full error message
-            startDate = p.dateStarted;
+          if (
+            !startDate ||
+            getValidDate(p?.dateStarted ?? "") < getValidDate(startDate)
```

Comment:
What happens when we compare undefined to something? Is it a safe operation?

Reply:
why would it be undefined? If dateStarted is missing, it defaults to today

Reply:
I missed that we were mutating the argument we passed in to assign today's date. My concern is not relevant here.